src/test/mocks/MockERC1155.sol

```diff
@@ -3,6 +3,6 @@ pragma solidity ^0.8.0;
 
 import "@openzeppelin/contracts/token/ERC1155/presets/ERC1155PresetMinterPauser.sol";
 
-contract ERC1155Permission is ERC1155PresetMinterPauser {
+contract MockERC1155 is ERC1155PresetMinterPauser {
```

Comment:
i was hoping we should have 2 versions of Mock available for us to use so we can have better test coverage. For ERC20, 721, 1155.
1) Permissioned, only minter can mint.
2) Permissionless, everyone can mint.